### PR TITLE
zypper: Add environment example

### PR DESCRIPTION
**Pull request by robinro**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
zypper.py

##### ANSIBLE VERSION
devel

##### SUMMARY
Add example for usage of ZYPP_LOCK_TIMEOUT

Retries to require lock for zypper operation.

### CODE REVIEW

File: lib/ansible/modules/packaging/os/zypper.py [hunk 1]
```diff
@@ -107,6 +107,13 @@
         required: false
         default: "no"
         choices: [ "yes", "no" ]
+    env:
+        version_added: "2.3"
+        description:
+            - Dict of env variables to pass to zypper. For example ZYPP_LOCK_TIMEOUT to wait for some time when tryping to acquire
```
Comment (gundalow) [inline lib/ansible/modules/packaging/os/zypper.py]: I think this needs to be on one line

```
2017-01-11 13:30:15 Run command: test/sanity/validate-modules/validate-modules lib/ansible/modules/packaging/os/zypper.py --exclude '^(lib/ansible/modules/utilities/logic/async_status.py|lib/ansible/modules/utilities/helper/_accelerate.py)'
2017-01-11 13:30:15 ============================================================================
2017-01-11 13:30:15 lib/ansible/modules/packaging/os/zypper.py
2017-01-11 13:30:15 ============================================================================
2017-01-11 13:30:15 TRACE:
2017-01-11 13:30:15     while scanning a simple key
2017-01-11 13:30:15       in "<string>", line 79, column 13:
2017-01-11 13:30:15                     a lock.
2017-01-11 13:30:15                     ^
2017-01-11 13:30:15     could not find expected ':'
2017-01-11 13:30:15       in "zypper.DOCUMENTATION", line 115, column 9:
2017-01-11 13:30:15                 required: false
2017-01-11 13:30:15                 ^
2017-01-11 13:30:15 ERROR: DOCUMENTATION is not valid YAML. Line 115 column 9
```

Comment (robinro) [reply lib/ansible/modules/packaging/os/zypper.py]: Thanks, fixed it.

**Comment by sivel** [comment]
I'm wondering why we wouldn't just recommend using the `environment` directive on the task instead of duplicating environment functionality into module args?

**Comment by robinro** [comment]
@sivel you're correct of course. I was thinking of too much other stuff to realize this. So this is downgraded to a docs pull request with an example of using the environment directive.

**Pull request merged**